pkg/controller/kccmanager/kccmanager.go

```diff
@@ -185,8 +187,20 @@ func New(ctx context.Context, restConfig *rest.Config, cfg Config) (manager.Mana
 	}
 	rd.DependencyTracker = gcpwatch.NewDependencyTracker(fetcher)
 
+	pollInterval := 10 * time.Minute
```

Comment:
Can we move the default poll interval value to `tracker.go`? I think it would be clearer if all default values for `PollConfig` are defined in the same file as `PollConfig`.